mypyc/external/googletest/include/gtest/gtest.h

```diff
@@ -1287,7 +1287,7 @@ class GTEST_API_ UnitTest {
   internal::UnitTestImpl* impl() { return impl_; }
   const internal::UnitTestImpl* impl() const { return impl_; }
 
-  // These classes and funcions are friends as they need to access private
+  // These classes and functions are friends as they need to access private
```

Comment:
should I exclude `googletest` , I guess it is some vendored code, isn't it?

Reply:
I don't know much about mypyc, but digging through the commit history: yeah, it's vendored, and hasn't been changed in 4 years. Probably best left alone.